src/ior.h

```diff
@@ -100,6 +100,7 @@ typedef struct
     int numTasks;                    /* number of tasks for test */
     int nodes;                       /* number of nodes for test */
     int tasksPerNode;                /* number of tasks per node */
+    int packedTasks;                 /* are the tasks round-robin across nodes or are they packed tightly*/ 
```

Comment:
maybe instead of "packed" the term mpiBlockMapping would be more appropriate?

Reply:
Nice suggestion.  Changed.